spec/rails_helper.rb

```diff
@@ -73,6 +73,8 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
To use https://api.rubyonrails.org/v5.2.4/classes/ActiveSupport/Testing/TimeHelpers.html instead of Timecop helpers: https://github.com/dradis/dradis-ce/pull/700#discussion_r972857626